dsl/frontend/frontend.ml

```diff
@@ -0,0 +1,3 @@
+module Ast = Ast
+module Parser = Parse
```

Comment:
Sorta smells like one of these names should just be changed? What do you think

Reply:
Resolved through synchronous discussion.

We'd like to have both the LHS and RHS say `Parser`, but we can't name our `Parse` module that because `menhir` already generates a module called `Parser` :(